instructor/client_anthropic.py

```diff
@@ -35,9 +35,10 @@ def from_anthropic(
         | anthropic.AnthropicBedrock
         | anthropic.AsyncAnthropicBedrock
         | anthropic.AsyncAnthropicVertex
+        | anthropic.AnthropicVertex
```

Comment:
Presentin an overload, so we needed this here.

Reply:
+1